external-dependencies.gradle

```diff
@@ -15,11 +15,9 @@ ext {
             "io.dropwizard.metrics:metrics-graphite:${dropwizard_metrics_version}"
     ]
 
-    dropwizardLogstash = "uk.gov.ida:dropwizard-logstash:1.1.2-build_45"
+    dropwizardLogstash = "uk.gov.ida:dropwizard-logstash:1.2.2-SNAPSHOT"
```

Comment:
please don't use `SNAPSHOT` dependencies.  If you want a later version you can use `1.2.0-build_50`